### PR TITLE
Fixed Navbar Issue. 

### PR DESCRIPTION
**Pull request by tusharrao198**
1. Added animations on Navbar.
2. Updated team_style.css ( Removed class bg-light from team div container tag because of  odd color combination , probably not noticable but yes.)

### CODE REVIEW

File: src/stac_iitmandi/static/stac_iitmandi/css/style.css [hunk 1]
```diff
@@ -34,32 +48,285 @@ hr {
...
+    left: 0;
+    height: 2px;
+    transition: all 500ms ease;
+}
+
+@media (min-width: 768px) {
+    .navbar-b.navbar-trans .nav-link:before,
+    .navbar-b.navbar-reduce .nav-link:before {
+        margin-left: 18px;
+    }
```
Comment (LalitNM) [inline src/stac_iitmandi/static/stac_iitmandi/css/style.css:113]: Including this, all media queries can be shifted to the bottom of the file.

File: src/stac_iitmandi/templates/stac_iitmandi/template.html [hunk 2]
```diff
@@ -1,8 +0,0 @@
-{% extends 'stac_iitmandi/base.html' %}
-{% block content %}
-<div class="container bg-light mt-2">
-  <h1>Heading here</h1>
-  <p>This is dummy text Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
-    consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-</div>
-{% endblock content%}
```
Comment (LalitNM) [inline src/stac_iitmandi/templates/stac_iitmandi/template.html:8]: Can you tell me a reason of deleting this file. In my opinion this should be there for future references.

**Comment by gajrajgchouhan** [comment]
Why was `/core_team/abhijeet.jpg`  deleted ?

**Comment by tusharrao198** [comment]
> Why was `/core_team/abhijeet.jpg` deleted ?

Former coordinators are not added in the core team wiki page.

**Comment by LalitNM** [comment]
> > Why was `/core_team/abhijeet.jpg` deleted ?
> 
> Former coordinators are not added in the core team wiki page.

Please add them :sweat_smile:, they are part of team.

**Comment by tusharrao198** [comment]
> > > Why was `/core_team/abhijeet.jpg` deleted ?
> > 
> > 
> > Former coordinators are not added in the core team wiki page.
> 
> Please add them 😅, they are part of team.

Issue #174 has been created for this. In this way you can understand of adding users from the django-admin section.

**Comment by LalitNM** [comment]
Great @tusharrao198 :clap: , merging this.

File: src/stac_iitmandi/static/stac_iitmandi/css/style.css (hunk 1 above)
Comment (LalitNM) [reply src/stac_iitmandi/static/stac_iitmandi/css/style.css:113]: New issue will be created.

File: src/stac_iitmandi/templates/stac_iitmandi/template.html (hunk 2 above)
Comment (LalitNM) [reply src/stac_iitmandi/templates/stac_iitmandi/template.html:8]: New issue will be created.

**Pull request merged**